app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ActiveRecord::Base
+  has_attached_file :image, styles: { large: "600x600>", medium: "300x300>", thumb: "150x150#" }
+  validates :title, presence: true
+  validates :content, presence: true
+  validates :image, attachment_presence: true
+  validates_attachment_content_type :image, content_type: /\Aimage\/.*\Z/
+  validates_with AttachmentSizeValidator, attributes: :image, less_than: 4.megabytes
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)